AC3.2Unit2FinalAssessment/ColorTableViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ColorTableViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Simone Grant on 10/6/16.
+//  Copyright © 2016 Simone Grant. All rights reserved.
+//
+
+import UIKit
+
+class ColorTableViewController: UITableViewController {
+    
+   var data = [crayolaColors]
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+            for c in crayolaColors {
+                if let crayon = Crayon(fromDict: c) {
+//                    crayon.append(crayon)
```

Comment:
`crayons.append(crayon)` this and the above code would have given you the correct data model.